### DISCUSSION TITLE
[PNUPC] medium 난이도까지 풀어봤습니다!

### DISCUSSION THREAD

**Discussion by pykido** [BDD-CLUB/challenge-every-week-posting#499] [4. 기타 글 모음]
- 5월 4일에 열린 PNUPC 기출 문제들이 백준에 올라왔길래 Medium 난이도까지 풀어봤습니다!
1. [백준 31789 모험의 시작](https://animoto1.tistory.com/entry/%EB%B0%B1%EC%A4%80-31789-%EB%AA%A8%ED%97%98%EC%9D%98-%EC%8B%9C%EC%9E%91-C)
2. [백준 31797 아~파트 아파트 ](https://animoto1.tistory.com/entry/%EB%B0%B1%EC%A4%80-31797-%EC%95%84%ED%8C%8C%ED%8A%B8-%EC%95%84%ED%8C%8C%ED%8A%B8-C)
3. [백준 31796 한빛미디어(Easy)](https://animoto1.tistory.com/entry/%EB%B0%B1%EC%A4%80-31796-%ED%95%9C%EB%B9%9B%EB%AF%B8%EB%94%94%EC%96%B4-Easy-C)
4. [백준 31791 바이러스 공격](https://animoto1.tistory.com/entry/%EB%B0%B1%EC%A4%80-31791-%EB%B0%94%EC%9D%B4%EB%9F%AC%EC%8A%A4-%EA%B3%B5%EA%B2%A9-C)

- 바이러스 공격 문제가 어려워 시간을 꽤 많이 썼습니다 ㅠㅠ..
- 종만북 다 읽을 때쯤 Hard 난이도 또한 도전해보겠습니다!
![KakaoTalk_Photo_2024-05-12-15-09-57](https://github.com/BDD-CLUB/challenge-every-week-posting/assets/77539625/89031861-230d-4f1f-b269-f6af4fe36276)

**Comment by jojaeng2** [comment]
오.. 이런게 있었군요 저도 풀어봐야겠네요

**Comment by stopmin** [comment]
대단합니닷 !!
